tools/test.sh

```diff
@@ -2,12 +2,16 @@
 
 # Copyright (c) 2018-2020 MobileCoin Inc.
 
+# RUSTFLAGS="-C instrument-coverage" \
```

Comment:
delete comment

Reply:
gonna leave that in for reference in the future